index.js

```diff
@@ -3,10 +3,12 @@ module.exports = {
     es6: true,
     node: true,
     browser: true,
-    jest: true,
-    mocha: true
+    jest: true
```

Comment:
Why the decision to take out mocha here? Is it because mocha is used in the truffle preset? What if somebody wants to use mocha in a js project that doesn't use truffle? Or are we assuming jest will be the default because truffle was the only reason we'd be using mocha in the first place?

Reply:
Exactly! Jest is the default because it's amazing. I'd use Jest for truffle tests if it wasn't super janky.

I left it in assuming that this would work but forgot of all the other global things it needed too.